addons/mail/models/mail_mail.py

```diff
@@ -83,6 +83,24 @@ def default_get(self, fields):
     scheduled_date = fields.Char('Scheduled Send Date',
         help="If set, the queue manager will send the email after the date. If not set, the email will be send as soon as possible. Unless a timezone is specified, it is considered as being in UTC timezone.")
 
+    @api.model
+    def fields_get(self, *args, **kwargs):
+        # related selection will fetch translations from DB
+        # selections added in stable won't be in DB -> add them on the related model if not already added
+        message_type_field = self.env['mail.message']._fields['message_type']
+        if ('auto_comment', 'Automated Targeted Notification') not in message_type_field.get_description(self.env)['selection']:
```

Comment:
Maybe we should check label only to be resilient against description ?
```suggestion
        if 'auto_comment' not in {label for label, _desc in message_type_field.get_description(self.env)['selection']}:
```